modules/dists/dims/BlockDim.chpl

```diff
@@ -27,7 +27,8 @@ private use DimensionalDist2D;
 
 /*
 This Block dimension specifier is for use with the
-``dimensionalDist2D`` distribution.
+:mod:`dimensionalDist2D <DimensionalDist2D>` distribution.
+:mod:`DimensionalDist2D` distribution.
```

Comment:
Looks like there is an extra line here

Reply:
🤦 

Funny, it wasn't there in my (unsaved) editor buffer!

Thanks for catching that...